README.md

```diff
@@ -83,3 +121,28 @@ public class PluginLogic implements RealtimePluginInterface<PluginSettings> {
 
```

Comment:
It would be nice to have a gradle task for this. It would help make the command line use platform-independent.